core/state_processor.go

```diff
@@ -61,10 +60,7 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 		allLogs  []*types.Log
 		gp       = new(GasPool).AddGas(block.GasLimit())
 	)
-	// Mutate the block and state according to any hard-fork specs
```

Comment:
This was removed to prevent an irregular state transition being applied at the daofork height.